frontend/src/plugins/router.js

```diff
@@ -51,4 +52,9 @@ const router = createRouter({
   routes,
 });
 
+router.afterEach(() => {
+  // Scroll to top to avoid annoying behaviour in mobile
+  window.scrollTo({ top: 0, left: 0 });
```

Comment:
Hope this doesn't break anything lol

Reply:
I tested it a lot and seems to work as intended. Crossed fingers 🤣